BCStudentSoftwareDevTeam/celts
# Order event list 205

# This PR fixes issue #205 

## Summary: 
We have adjusted a back-end query that will now get whichever event is linked to the most programs in a given term. It will take the id of that link and another query that grabs all training will ensure that when the id of all training appears it will be first in the object and from there on will order by descending start date.

## To Test: 
You can check this out by getting the term for an all training event and switching over to the training tab in that term and you will see that all training appears first and afterward the events will appear by descending dates.

## Files changed: 
`logic/events`

## UI:
![image](https://user-images.githubusercontent.com/67805799/163583900-efeb3675-54a0-45f0-b568-4d78af3c9d79.png)
